tests/third_party/cupy/fft_tests/test_fft.py

```diff
@@ -317,39 +317,49 @@ def test_irfftn(self, dtype):
                 xp.fft.irfftn(a, s=self.s, axes=self.axes, norm=self.norm)
 
 
+@pytest.mark.usefixtures("skip_forward_backward")
 @testing.parameterize(
     *testing.product(
         {
             "n": [None, 5, 10, 15],
             "shape": [(10,), (10, 10)],
-            "norm": [None, "ortho"],
+            "norm": [None, "backward", "ortho", "forward", ""],
         }
     )
 )
-@pytest.mark.usefixtures("allow_fall_back_on_numpy")
 class TestHfft:
     @testing.for_all_dtypes()
     @testing.numpy_cupy_allclose(
         rtol=1e-4,
-        atol=1e-7,
+        atol=2e-6,
+        accept_error=ValueError,
+        contiguous_check=False,
         type_check=has_support_aspect64(),
```

Comment:
Will the test work fin with stock numpy? Or we need to use `type_check=False` here for that?

Reply:
It will work. Data type for `NumPy` case is accordingly updated (similar to `cupy`) on line 344.